src/read/op.rs

```diff
@@ -705,7 +705,11 @@ where
             }
             constants::DW_OP_stack_value => Ok(Operation::StackValue),
             constants::DW_OP_implicit_pointer | constants::DW_OP_GNU_implicit_pointer => {
-                let value = bytes.read_offset(encoding.format)?;
+                let value = if encoding.version != 2 {
```

Comment:
Please change the test to `==` and swap the branches. Same for the if expression for writing.

Reply:
Swapped